int16 servo_id
string status
string mode
